common/logisticspipes/utils/FluidSinkReply.java

```diff
@@ -0,0 +1,16 @@
+package logisticspipes.utils;
+
+public final class FluidSinkReply {
+    public enum FixedFluidPriority{
+        Terminus,
+        ItemSink
```

Comment:
use regular Java naming scheme: all uppercase "TERMINUS" and "ITEM_SINK" and add members for translation keys, if applicable